### DISCUSSION TITLE
Welcome to the 3rd phase

### DISCUSSION THREAD

**Discussion by dhakelila** [Vizzuality/mangrove-atlas#287] [General]
Hello everyone, 

As agreed during the Kick-off, we'll be moving all the conversations to Github issues. 
From now on, anything you need to discuss, please, continue an existing thread or open a new one. 

Cheers!

**Comment by dhakelila** [comment]
Just for reference, I'm pinning here the old Basecamp link. 
https://basecamp.com/1756858/projects/16449126

**Comment by lyubovpencheva** [comment]
this is old one
